src/stores/widgets/StopGapWidget.ts

```diff
@@ -445,6 +450,12 @@ export class StopGapWidget extends EventEmitter {
         this.feedEvent(ev);
     };
 
+    private onToDeviceEvent = async (ev: MatrixEvent) => {
```

Comment:
Do to-device-events work differently than other events? I am asking because when an encrypted event arrives at `onEvent` decryption is triggered. After encryption `onEvent` will be called again. Can this happen here, that `onToDeviceEvent` is called again after `await this.client.decryptEventIfNeeded(ev);`?